src/main/java/org/tikv/common/PDClient.java

```diff
@@ -143,7 +145,7 @@ public TiTimestamp getTimestamp(BackOffer backOffer) {
    *
    * @param region represents a region info
    */
-  void scatterRegion(TiRegion region, BackOffer backOffer) {
+  void scatterRegion(Metapb.Region region, BackOffer backOffer) {
```

Comment:
public api change may cause compatibility issues

Reply:
Looks like this is a package protected method that code within TiKV client can access only. We can be sure all references to this method are fixed as well.